examples/balances.ts

```diff
@@ -9,17 +9,22 @@ import path from "path";
 const rootPath = path.resolve();
 dotenv.config({ path: path.join(rootPath, ".env") });
 
+// Script config for ganache
+const ETHEREUM_PRIVATE_KEY_DEFAULT =
+  "0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e";
```

Comment:
Is not better bring this from .env?

Reply:
You're right, is the ganache key from nf3 though... Hopefully #71 should save us from this nightmare :see_no_evil: